assignments/callbacks.js

```diff
@@ -20,28 +20,53 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
   });
 
 */
-
+// getLength passes the length of the array into the callback.
 
 function getLength(arr, cb) {
-  // getLength passes the length of the array into the callback.
+      
+  return cb(arr.length);
 }
+getLength(items, function(length) {
+console.log(length);
+});
 
-function last(arr, cb) {
-  // last passes the last item of the array into the callback.
+// last passes the last item of the array into the callback.
+function last(array, callback) {
+  callback(array[array.length - 1]);
 }
+last(items, (last) => {
+  console.log(last);
+});    
+  
+// sumNums adds two numbers (x, y) and passes the result to the callback.
+ 
 
 function sumNums(x, y, cb) {
-  // sumNums adds two numbers (x, y) and passes the result to the callback.
+  cb(x + y);
 }
+sumNums(4, 5, (total) => {
+  console.log(total);
+});
+// multiplyNums multiplies two numbers and passes the result to the callback.
 
 function multiplyNums(x, y, cb) {
-  // multiplyNums multiplies two numbers and passes the result to the callback.
+  cb(x * y);
 }
+multiplyNums(4,5, (sum) => {
+  console.log(sum);
+});
 
+// contains checks if an item is present inside of the given array/list.
+// Pass true to the callback if it is, otherwise pass false.
 function contains(item, list, cb) {
-  // contains checks if an item is present inside of the given array/list.
-  // Pass true to the callback if it is, otherwise pass false.
+
+  cb(item === list);
+  return "True";
 }
+  contains("Pencil",'Pencil', (comp) => {
```

Comment:
above you have item, list, cc - here you should have "Pencil", items, cbfunction... does that make sense? I would encourage looking at the solution to compare and contrast.